specs/latest/1.0/index.html

```diff
@@ -1690,9 +1689,8 @@ <h2 class="no-toc">Table of contents</h2>
                            GLenum srcAlpha, GLenum dstAlpha);
 
     void bufferData(GLenum target, GLsizeiptr size, GLenum usage);
-    void bufferData(GLenum target, ArrayBuffer? data, GLenum usage);
-    void bufferData(GLenum target, ArrayBufferView data, GLenum usage);
-    void bufferSubData(GLenum target, GLintptr offset, BufferDataSource data);
+    void bufferData(GLenum target, [AllowShared] BufferSource? data, GLenum usage);
```

Comment:
Indeed, `null` is a valid value of type `[AllowShared] BufferSource? data`, so it'll still end up hitting the prose which causes the error.